### PR TITLE
Issue_3389 - Upgrade documentation updates

### PR DESCRIPTION
**Pull request by dlarson04**
Signed-off-by: Doug Larson <larsond@us.ibm.com>

# Pull Request Template

## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Documentation updates 

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Peer reviewed

## Additional Context (Please include any Screenshots/gifs if relevant)

...

## Checklist:

- [ x] My code follows the style guidelines of this project
- [ x] I have performed a self-review of my own code
- [ x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [x ] I have checked my code and corrected any misspellings
- [ x] I have tagged the reviewers in a comment below incase my pull request is ready for a review
- [ x] I have signed the commit message to agree to Developer Certificate of Origin (DCO) (to certify that you wrote or otherwise have the right to submit your contribution to the project.) by adding "--signoff" to my git commit command.

### CODE REVIEW

File: docs/node_management_overview.md [hunk 1]
```diff
@@ -2,67 +2,35 @@
...
-                "agent-install.cfg"
-            ],
-            "version": "latest"
-        }
-    }
-    ```
-    **Note**: The files and versions must correspond to objects stored on the Management Hub. Use the `hzn nodemanagement agentfiles list` command to get a list of available files and versions.
-
-3. Add the manifest file to the Management Hub
+This capability utilizes existing agent artifacts that reside in Cloud Sync Service (CSS) from the edgeNodeFiles.sh installation script.
```
Comment (tbsloan) [inline docs/node_management_overview.md:7]: Should we change capability to feature here.  In 5 above, it states 'The Automatic Agent Upgrade is a policy-based node management feature ... ' 
For consistency, it reads better if we say 'This feature utilizes... '

File: docs/node_management_overview.md [hunk 2]
```diff
@@ -74,23 +42,24 @@ The Automatic Agent Upgrade is a policy-based node management feature that allow
...
+            "manifest": "<manifestID>",    # Substitute the actual manifestID from the available manifests
             "allowDowngrade": false
         }
     }
     ```
 
-6. Add the NMP to the Exchange
-    - Once the NMP is populated with all the necessary information, the next step is to add the NMP to the Exchange using the command below. In this example, the name of the NMP in the Hub will be `sample_nmp` and the file that contains the NMP is called `nmp.json` 
+3. Verify the impacted edge nodes (Optional)
+    The following command can be used to check which nodes the NMP will deploy to before publishing the NMP to the Exchange. This is useful to ensure the NMP does not deploy to any unintended nodes.
```
Comment (tbsloan) [inline docs/node_management_overview.md:52]: Step 3 - let's change '...NMP will deploy to..' 
to 
'...NMP applies to..'

**Review by kmbarton423 — APPROVED** [review]
Looks great.

**Review by tbsloan — APPROVED** [review]
Doug - looks good to me.

**Pull request merged**